### DISCUSSION TITLE
Telegram bot 错误信息解决方案汇总

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2919] [Developer]
> 以下是调用失败的一些说明：

***

![image](https://user-images.githubusercontent.com/81074/136157787-949dc183-5ee6-403d-9bf4-8073f9222a55.png)

> 重新使用 `/token 你的 Token` 绑定即可。

***

![image](https://user-images.githubusercontent.com/81074/122329272-b298c380-cf63-11eb-9dd1-6096d372ddba.png)

> 设置 Telegram ID 即可解决，细节 [请看这里](https://github.com/Kenshin/simpread/discussions/2399)。

***

![image](https://user-images.githubusercontent.com/81074/136158000-6631d5dd-b1ee-4c7e-9a94-e4659e8c594d.png)

> 说明通过授权服务的信息有误，需要重新绑定授权服务

**坚果云方案**

1. 扩展端 → 服务 → 授权管理 → **取消坚果云的授权，再重新授权**

![image](https://user-images.githubusercontent.com/81074/136158249-083627d3-6e90-4913-b001-fe056c76a4fa.png)

2. 重新授权登录

![image](https://user-images.githubusercontent.com/81074/136158306-b24083e5-e25a-49c0-bb1a-2541750c954d.png)

**Dropbox 方案**

1. 扩展端 → 服务 → 授权管理 → **取消坚果云的授权，再重新授权**

![image](https://user-images.githubusercontent.com/81074/136158516-0b989543-1490-4b76-9d68-2599a730fc0a.png)

2. 重新授权登录

![image](https://user-images.githubusercontent.com/81074/136158409-e9cbc409-2d3b-435c-b160-f3226eb72103.png)

***

> 如果上述仍无法解决你的问题，请看这里：

- https://github.com/Kenshin/simpread/discussions/2281

- https://github.com/Kenshin/simpread/discussions/2282

**Comment by Joel2561** [comment]
![122329272-b298c380-cf63-11eb-9dd1-6096d372ddba](https://user-images.githubusercontent.com/77143461/142373858-ebb40515-1f98-4d27-900f-74fd362935fa.png)
关于这一条，也有可能是在开放平台中删除或更改了API导致的。
